[ID1000323] increase the type of users for charities

Nixiangyu opened the pull request:
--story=1000323 --user=Nixiangyu
https://www.tapd.cn/60210943/prong/stories/view/1160210943001000323

Signed-off-by: Nixiangyu <nixiangyu@ofworld.com>

Commit:
[ID1000323] increase the type of users for charities

--story=1000323 --user=Nixiangyu
https://www.tapd.cn/60210943/prong/stories/view/1160210943001000323

Signed-off-by: Nixiangyu <nixiangyu@ofworld.com>

Commit:
[ID1000323] increase the type of users for charities

--story=1000323 --user=Nixiangyu
https://www.tapd.cn/60210943/prong/stories/view/1160210943001000323

Signed-off-by: Nixiangyu <nixiangyu@ofworld.com>

Commit:
[ID1000323] increase the type of users for charities

--story=1000323 --user=Nixiangyu
https://www.tapd.cn/60210943/prong/stories/view/1160210943001000323

Signed-off-by: Nixiangyu <nixiangyu@ofworld.com>

Commit:
[ID1000323] increase the type of users for charities

--story=1000323 --user=Nixiangyu
https://www.tapd.cn/60210943/prong/stories/view/1160210943001000323

Signed-off-by: Nixiangyu <nixiangyu@ofworld.com>

zhangbing16 commented on router/router.go:
```diff
@@ -45,8 +45,8 @@ const (
 	// org
-	urlOrgsQuery     = "orgs/query"
-	urlOrgsInfoQuery = "orgs/detail/query"
+	urlOrgCharitiesQuery  = "org/charities/query"
```
query是动词，可以去掉，表示查询列表

Commit:
[ID1000323] increase the type of users for charities

--story=1000323 --user=Nixiangyu
https://www.tapd.cn/60210943/prong/stories/view/1160210943001000323

Signed-off-by: Nixiangyu <nixiangyu@ofworld.com>

Commit:
[ID1000323] increase the type of users for charities

--story=1000323 --user=Nixiangyu
https://www.tapd.cn/60210943/prong/stories/view/1160210943001000323

Signed-off-by: Nixiangyu <nixiangyu@ofworld.com>
